apps/chat/src/components/Common/ApplicationWizard/ApplicationWizardFooter.tsx

```diff
@@ -96,25 +100,20 @@ export const ApplicationWizardFooter: FC<ApplicationWizardFooterProps> = ({
             <Tooltip tooltip={t('Delete')}>
               <button
                 onClick={handleConfirmDialogOpen}
-                className="flex size-[34px] items-center justify-center rounded text-secondary hover:bg-accent-primary-alpha hover:text-accent-primary"
+                className="icon-button"
                 data-qa="application-delete"
+                disabled={isAppDeployed}
```

Comment:
for deploying and undeploying also should be disabled (`isApplicationStatusUpdating`)